<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>portfolio_plot</class>
 <widget class="QMainWindow" name="portfolio_plot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1743</width>
    <height>1291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Portfolio Stats</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>1150</y>
      <width>1711</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item>
      <widget class="QPushButton" name="refresh_b">
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stats_btn">
       <property name="text">
        <string>Stats</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="b_close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1711</width>
      <height>1131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout"/>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout_right"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1743</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
